hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/src/main/java/org/hawkular/alerts/actions/email/EmailPlugin.java

```diff
@@ -302,9 +302,9 @@ protected Message createMimeMessage(ActionMessage msg) throws Exception {
             if (alert.getStatus().equals(Status.OPEN)) {
                 email.setSentDate(new Date(alert.getCtime()));
             } else if (alert.getStatus().equals(Status.ACKNOWLEDGED)) {
-                email.setSentDate(new Date(alert.getAckTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             } else {
-                email.setSentDate(new Date(alert.getResolvedTime()));
+                email.setSentDate(new Date(alert.getCurrentLifecycle().getStime()));
             }
```

Comment:
By the way,
-  alert.getAckTime
-  alert.getResolvedTime

can be ambiguous so I guess we can use 
-  alert.getLastAckTime
-  alert.getLastResolvedTime

as we introduced getCurrent* terminology getLast* makes sense to me.